### PR TITLE
Bugfix/ees 769 amendments create release subject migration

### PR DESCRIPTION
**Pull request by shakes90**

### CODE REVIEW

File: src/GovUk.Education.ExploreEducationStatistics.Data.Model/Footnote.cs [hunk 1]
```diff
@@ -7,13 +7,15 @@ public class Footnote
...
-        public Guid ReleaseId { get; set; }
+
         public ICollection<IndicatorFootnote> Indicators { get; set; }
         public ICollection<FilterFootnote> Filters { get; set; }
         public ICollection<FilterGroupFootnote> FilterGroups { get; set; }
         public ICollection<FilterItemFootnote> FilterItems { get; set; }
         public ICollection<SubjectFootnote> Subjects { get; set; }
+
+        // TODO BAU-384 - remove when this work goes out
+        public Release Release { get; set; }
```
Comment (benoutram) [inline src/GovUk.Education.ExploreEducationStatistics.Data.Model/Footnote.cs:18]: Is this change intentional?

Comment (shakes90) [reply src/GovUk.Education.ExploreEducationStatistics.Data.Model/Footnote.cs:18]: yep - it got accidentally removed prematurely by dunc - will go ultimately.
